x-pack/plugins/security_solution/common/search_strategy/ecs_fields/extend_map.test.ts

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
what do you think about moving all of that in the folder of `security_solution/ecs`